### PR TITLE
CLDR-17803 site: popup menu for subpages

### PR DESCRIPTION
**Pull request by srl295**
CLDR-17803

'popup' menu of subpages

image

image

@macchiati with position:absolute the menu was harder to access (it would disappear when trying to move over to it

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/site/assets/css/page.css [hunk 1]
```diff
@@ -43,20 +43,36 @@ header .nav ul b {
...
+    padding: 0.5３m；
 }
 
 header .nav ul li {
-    display: inline-flex;
+    display: block;
     padding: .5em;
+    font-size: medium;
     margin: 0;
+    /* color: black !important; */
```
Comment (btangmu) [inline docs/site/assets/css/page.css:75]: better not to add commented-out code

File: docs/site/assets/js/cldrsite.js [hunk 2]
```diff
@@ -215,6 +218,8 @@ const app = Vue.createApp(
                 </a>
             </li>
         </ul>
+  </div>
```
Comment (btangmu) [inline docs/site/assets/js/cldrsite.js:221]: needs formatting

**Review by btangmu — DISMISSED** [review]

**Comment by srl295** [comment]
@btangmu take a look now

**Comment by srl295** [comment]
> Tried out https://faa11b76.cldr.pages.dev/. Looks great

can i get a review on this one?

**Review by macchiati — APPROVED** [review]

**Pull request merged**